torch/_inductor/config.py

```diff
@@ -787,6 +787,13 @@ class cuda:
     # epilogue fusion via CUTLASS Epilogue Visitor Trees ( EVT )
     # are enabled for the CUTLASS backend.
     cutlass_only_evt_capable_ops: bool = False
+    # Keep only Cutlass op configs which contain this regular expression pattern
+    # Set this to "warpspecialized_cooperative_epi_tma" to enable only SM90 TMA Cutlass Kernels for large GEMMs
+    cutlass_op_whitelist_regex: Optional[str] = None
+
+    # Filter Cutlass configs which contain this regular expression pattern
+    # Set this to "pingpong" to avoid nondeterministic numerical issues
+    cutlass_op_blacklist_regex: Optional[str] = "pingpong"
```

Comment:
Change whitelist/blacklist to allowlist/denylist.

https://www.splunk.com/en_us/blog/learn/blacklist-whitelist-inclusivity.html

Reply:
done